Dokan/Dokan/Scenes/ProductDetails/CollectionViewDataSource.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  TableViewDataSource.swift
+//  Dokan
+//
+//  Created by heba isaa on 31/07/2022.
+//
+
+import Foundation
+import UIKit
+
+class CollectionViewDataSource: NSObject, UICollectionViewDataSource {
```

Comment:
Please check how the reviews view is suggested [here](https://github.com/motoon-eg/dokan-store/pull/65/files#r939449480) and apply a similar approach.